simtools/simtel/simulator_light_emission.py

```diff
@@ -398,6 +354,26 @@ def remove_line_from_config(file_path, line_prefix):
 
         return command
 
+    def _remove_line_from_config(self, file_path, line_prefix):
```

Comment:
Do you have a unit test for this function? I didn't see it.

Reply:
Thanks for pointing that out. A test is included now.